ui/admin/app/templates/scopes/scope.hbs

```diff
@@ -6,13 +6,43 @@
 {{#if @model.isOrg}}
   {{page-title (t 'resources.org.title_plural')}}
   {{page-title @model.displayName}}
+  <Breadcrumbs::Item
+    @text={{t 'resources.org.title_plural'}}
+    @route='scopes.scope.scopes'
+    @model='global'
+  />
+  <Breadcrumbs::Item
+    @text={{@model.displayName}}
+    @route='scopes.scope.edit'
+    @model={{@model.id}}
+  />
 {{/if}}
 
 {{#if @model.isProject}}
   {{page-title (t 'resources.org.title_plural')}}
   {{page-title this.scopes.selectedOrg.displayName}}
   {{page-title (t 'resources.project.title_plural')}}
   {{page-title @model.displayName}}
+  <Breadcrumbs::Item
```

Comment:
Is this the only way to display these breadcrumbs? How did it work previously instead of us manually enumerating the scope breadcrumbs?

Reply:
This is how it worked before: https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/components/bread-crumbs.hbs  
https://github.com/hashicorp/boundary-ui/blob/main/ui/admin/app/controllers/scopes/scope/scopes.js